package.json

```
@@ -1,34 +1,89 @@
+    "vite": "^5.4.7",
+    "vite-plugin-environment": "^1.1.3",
+    "web-vitals": "^4.2.3"
   },
   "scripts": {
-    "serve": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "serve": "cross-env ESLINT_NO_DEV_ERRORS=true vite --config config/vite.config.ts",
+    "build": "tsc && vite build --config config/vite.config.ts",
+    "preview": "vite preview --config config/vite.config.ts",
+    "test:vitest": "vitest run",
```

Biome (1.9.4)

[error] The key test:vitest was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)